test/query/shorthand.test.ts

```diff
@@ -297,34 +427,19 @@ describe('query/shorthand', () => {
       assert.equal(str, 'a,q,sort={"field":"a","op":"mean","order":"descending"}');
     });
 
-    it('should return correct fieldDefShorthand string for bin with sort field definition object', () => {
-      const str = fieldDefShorthand({
-        bin: true, channel: Channel.X, field: 'a', type: Type.QUANTITATIVE, sort: {field: 'a', op: AggregateOp.MEAN, order: SortOrder.DESCENDING}
-      });
-      assert.equal(str, 'bin(a,q,sort={"field":"a","op":"mean","order":"descending"})');
-    });
-
-    it('should return correct fieldDefShorthand string for bin with maxbins and sort field definition object', () => {
-      const str = fieldDefShorthand({
-        bin: {maxbins: 20}, channel: Channel.X, field: 'a', type: Type.QUANTITATIVE, sort: {field: 'a', op: AggregateOp.MEAN, order: SortOrder.DESCENDING}
-      });
-      assert.equal(str, 'bin(a,q,maxbins=20,sort={"field":"a","op":"mean","order":"descending"})');
-    });
-
     it('should return correct fieldDefShorthand string for bin with maxbins, scale with scaleType ' +
        'and sort field definition object', () => {
       const str = fieldDefShorthand({
-        bin: {maxbins: 20}, channel: Channel.X, field: 'a', type: Type.QUANTITATIVE, scale: {type: ScaleType.LOG}, sort: {field: 'a', op: AggregateOp.MEAN, order: SortOrder.DESCENDING}
+        bin: {maxbins: 20}, channel: Channel.X, field: 'a', type: Type.QUANTITATIVE, scale: {type: ScaleType.LINEAR}
```

Comment:
Now the test description says sort but the test doesn't contain sort.   Update the description accordingly.  
Also make sure we still have a case where we test ordinal scale with sort definition object.  I believe that's the original intention of this test case.